lib/facebook/messenger/server.rb

```diff
@@ -122,7 +144,7 @@ def trigger_events(events)
         events['entry'.freeze].each do |entry|
           # Facebook may batch several items in the 'messaging' array during
           # periods of high load.
-          entry['messaging'.freeze].each do |messaging|
+          entry['messaging'.freeze].try(:each) do |messaging|
```

Comment:
I changed this one because there are other types of "messages" that FB Messenger API can throw at your callbacks. Namely, I've started to work on another callback (for validating the setup of the webhook) - see https://github.com/pjpires/facebook-messenger/compare/master...pjpires:support-webhook-validation. But you're probably right about ActiveSupport. :)